airflow/include/composeyml.yml

```diff
@@ -33,7 +33,11 @@ services:
     ports:
       - 127.0.0.1:{{ .PostgresPort }}:5432
     volumes:
+      {{if .DuplicateImageVolumes}}
```

Comment:
trying to understand this change, because the actual volume definition is slightly above: https://github.com/astronomer/astro-cli/pull/1052/files#diff-4e4b970bc22596b97b95d621fee0b2293a0df07fa231494d2f54da904dd8be05R18-R22, and this is just us referring that volume, so trying to figure out where is the duplicate definition?